.github/CODEOWNERS

```diff
@@ -1,39 +1 @@
-* @DataDog/ruby-guild
-
-# Public Documentation
-/docs/Compatibility.md @DataDog/documentation
-/docs/GettingStarted.md @DataDog/documentation
-
-# Library
-/lib/datadog/appsec/ @DataDog/asm-ruby
-/lib/datadog/appsec.rb @DataDog/asm-ruby
-/lib/datadog/tracing/ @DataDog/tracing-ruby
-/lib/datadog/tracing/contrib/ @DataDog/apm-idm-ruby
-/lib/datadog/tracing.rb @DataDog/tracing-ruby
-/lib/datadog/opentelemetry/ @DataDog/tracing-ruby
-/lib/datadog/opentelemetry.rb @DataDog/tracing-ruby
-/lib-injection/ @DataDog/tracing-ruby
-/lib/datadog/profiling/ @DataDog/profiling-rb
-/lib/datadog/profiling.rb @DataDog/profiling-rb
-/ext/ @DataDog/profiling-rb
-
-# RBS signatures
-/sig/datadog/appsec/ @DataDog/asm-ruby
-/sig/datadog/appsec.rbs @DataDog/asm-ruby
-/sig/datadog/tracing/ @DataDog/tracing-ruby
-/sig/datadog/tracing/contrib/ @DataDog/apm-idm-ruby
-/sig/datadog/tracing.rbs @DataDog/tracing-ruby
-/sig/datadog/opentelemetry/ @DataDog/tracing-ruby
-/sig/datadog/opentelemetry.rbs @DataDog/tracing-ruby
-/sig/datadog/profiling/ @DataDog/profiling-rb
-/sig/datadog/profiling.rbs @DataDog/profiling-rb
-
-# Specs
-/spec/datadog/appsec/ @DataDog/asm-ruby
-/spec/datadog/tracing/ @DataDog/tracing-ruby
-/spec/datadog/tracing/contrib/ @DataDog/apm-idm-ruby
-/spec/datadog/tracing_spec.rb @DataDog/tracing-ruby
-/spec/datadog/opentelemetry/ @DataDog/tracing-ruby
-/spec/datadog/opentelemetry_spec.rb @DataDog/tracing-ruby
-/spec/datadog/profiling/ @DataDog/profiling-rb
-/spec/datadog/profiling_spec.rb @DataDog/profiling-rb
+* @Appboy/sidekiq-and-monolith-foundations @zachmccormick
```

Comment:
Zach can't be a CODEOWNER anymore

![image](https://github.com/user-attachments/assets/a244dfc1-52d7-40c1-97fd-d41cb7962000)